tests/integration-tests/tests/createami/test_createami/test_build_image/image.config.yaml

```diff
@@ -14,8 +14,10 @@ Build:
     Tags:
         - Key: dummyBuildTag
           Value: dummyBuildTag
+{% if os in ["alinux2", "alinux2023"] %}
```

Comment:
Why should we update when using AL2 and AL2023?

Reply:
It is optional. Just to increase the diversity of the test

Reply:
Ok, let's add a comment specifying that, otherwise in future we may think that there is some more complex reasoning behind the choice.

Reply:
Done